.changeset/khaki-cobras-trade.md

```diff
@@ -0,0 +1,5 @@
+---
+"@comet/admin": patch
+---
+
+Removed the `disabled` ClassKey in `TabScrollButtonClassKey`
```

Comment:
This should probably mention an alternative for those, that used that `classKey`. 
Maybe: 

```suggestion
Removed the `disabled` ClassKey in `TabScrollButtonClassKey`

Use the `:disabled` selector on `root` instead when styling the disabled state. 
```